### ISSUE TITLE
Issue when running

### ISSUE THREAD

**Issue by alezanott** [beele/homebridge-unifi-protect-camera-motion#98]
Since the last restart of my Raspberry i continue to get this error
```
 TypeError: Cannot read property 'connected' of undefined
    at Timeout._onTimeout (/usr/lib/node_modules/homebridge-unifi-protect-camera-motion/src/utils/mqtt.js:35:33)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7)
```
I'm running Node JS v14.18.2 - npm v6.14.15

What can cause this kind of error? @beele can you help me?

**Comment by udotrautmann** [comment]
Hello,

I ' m getting this error too.

Thanks for any help

**Comment by nstefanelli** [comment]
Seeing the same error on my side

**Comment by beele** [comment]
Should be fixed in 0.4.8
